File: 110A-NearlyLuckyNumber/110a-nearlyLuckyNumber.py
if __name__ == "__main__":
  k = input()
  n = [int(x) for x in k]

  lucky = 0
  for num in n:
      if num == 4 or num == 7:
          lucky += 1
  if lucky == 4 or lucky == 7:
      print("YES")
  else:
      print("NO")

File: 110A-NearlyLuckyNumber/110a-nearlyLuckyNumber.cpp
#include <iostream>
#include <cstring>
using namespace std;
 
int main(){
	string k;
	cin >> k;
	int int_arr[k.length()], lucky = 0;
	
	char arr[k.length()+1];
	strcpy(arr,k.c_str());
	
	for(int i = 0; i < k.length(); i++){
		int_arr[i] = (int)arr[i] - '0';
		if(int_arr[i] == 4 || int_arr[i] == 7)
			lucky++;
	}
	if(lucky == 4 || lucky == 7)
		cout << "YES";
	else
		cout << "NO";
}
